reaper/reaper.go

```diff
@@ -86,7 +86,7 @@ func (r *Reaper) servicesWithName(serviceName string) <-chan cloudfoundry.Servic
 	return output
 }
 
-func (r *Reaper) freeServicePlansOf(services <-chan cloudfoundry.Service) <-chan cloudfoundry.ServicePlan {
+func (r *Reaper) ServicePlansWithName(services <-chan cloudfoundry.Service, planName string) <-chan cloudfoundry.ServicePlan {
```

Comment:
This should be `servicePlansWithName` rather than `ServicePlansWithName` since we don't want to export this function outside of the `reaper` package.